ktor-network/ktor-network-tls/jvm/src/io/ktor/network/tls/TLSClientHandshake.kt

```diff
@@ -242,7 +242,7 @@ internal class TLSClientHandshake(
                     var position = 0
                     while (position < authoritiesSize) {
                         val size = packet.readShort().toInt() and 0xFFFF
-                        position += size
+                        position += size + 2 // short = 2 bytes for reading size 
```

Comment:
Please express the intention for `+ 2` in the code instead of the comment

Reply:
Changed :)